images/multus-daemonset.yml

```diff
@@ -119,7 +119,7 @@ spec:
       serviceAccountName: multus
       containers:
       - name: kube-multus
-        image: nfvpe/multus:latest
+        image: kisanth88/kmultusv6test:latest
```

Comment:
Also, I don't think this change to the image path is intended?

Reply:
Image change is a mistake, sorry, had updated my fork for testing and the image change accidentally got added to pull request.

We do have other K8S addons working with this, however I did not explicitly retest it with IPv4 due to other priorities. I will make an attempt to retest this as soon as I can. Our lab has been unstable due to some switching upgrades that have been going on. So our business priorities have taken precedence over this request. I am happy to withdraw the pull request for now until I can do some detailed testing if you guys would like; however this is an issue with the entrypoint.sh and multus working with IPv6 only K8S environment that needs to be addressed.

Reply:
Replying with an idea that I think makes more sense than my original change.

I will try to take a whack at this if I can find some time, but it may be more forward looking and flexible to do a check in the entrypoint.sh script for if $KUBERNETES_SERVICES_HOST is IPv6 and then have two variants of the section with comment starting with:

# Write a kubeconfig file for the CNI plugin.